proposals/extensions.md

```diff
@@ -166,18 +150,12 @@ extension_modifier
     | 'partial'
     | 'unsafe'
     | 'static'
-    | 'protected'
```

Comment:
It looks like the modifiers on a type are modified here. Did you really mean to do that?

Reply:
That was incorrect. Restored. Thanks